apps/dispatch/dispatch-1.py

```diff
@@ -145,8 +145,12 @@ def __init__( self ) :
 	def _run( self, args ) :
 
 		if not len( args["tasks"] ) :
-			IECore.msg( IECore.Msg.Level.Error, "gaffer dispatch", "No task nodes were specified" )
-			return 1
+			# fallback to nodes for backwards compatibility
+			if len( args["nodes"] ) :
+				IECore.msg( IECore.Msg.Level.Error, "gaffer dispatch", "Use the \"tasks\" parameter to specify the dispatchable nodes. The \"nodes\" parameter is only for gui purposes." )
```

Comment:
Warning rather than Error?

Reply:
Addressed in 565b5a60f05aedeed008ec1d77411d29c03fed96